crates/cheatnet/src/cheatcodes/deploy.rs

```diff
@@ -23,9 +23,11 @@ impl CheatnetState {
         &mut self,
         class_hash: &ClassHash,
         calldata: &[Felt252],
-        salt: ContractAddressSalt,
         contract_address: ContractAddress,
     ) -> Result<ContractAddress, CheatcodeError> {
+        let salt = self.get_salt();
+        self.increment_deploy_salt_base();
```

Comment:
I'm not sure why was this moved here. Won't it influence the contract address?

Reply:
I think it is ok to move it here. Previously `deploy_at` didn't change the salt so I think deploying two same contracts with the `deploy_at` function would panic. 
@piotmag769 could you please add an issue (or a test) for checking it?

Reply:
It is to avoid having to pass salt to `deploy_at` so it can be used directly from `forge` as salt is stored in cheatnet

I think we already have a test for that, don't we? `deploy_two_at_the_same_address` in cheatnet